voyager-rxjava/build.gradle.kts

```diff
@@ -1,28 +1,19 @@
 plugins {
-    kotlin("multiplatform")
+    kotlin("android")
```

Comment:
@Syer10 I saw the description but you can make it compatible using the implementation for android specifically. (implementation 'io.reactivex.rxjava3:rxandroid:3.0.2')

Check out this: [https://github.com/ReactiveX/RxAndroid](https://github.com/ReactiveX/RxAndroid)

Reply:
RxJava is platform agnostic and the specifics of the platform should be solved on the platform. With this I mean that this should be kept as multiplatform, the code on this module is not related to compose or any platform.

Reply:
Fixed by just using RxJava as a direct dependency instead of the Compose RxJava dependency

Reply:
Great! Looks good to me. Thanks! 🙇‍♂️